etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestrator.java

```diff
@@ -191,7 +190,6 @@ public Optional<PartnerMetadata> getMetadata(String receivedSubmissionId)
             var ourStatus = ourStatusFromReportStreamStatus(rsStatus);
 
             logger.logInfo("Updating metadata with receiver {} and status {}", receiver, ourStatus);
-            partnerMetadata = partnerMetadata.withReceiver(receiver).withDeliveryStatus(ourStatus);
```

Comment:
Did you mean to remove `.withDeliveryStatus(ourStatus)` here?

Reply:
No, added it back